chain/kusama/defaults.go

```diff
@@ -4,56 +4,53 @@
 package kusama
 
 import (
-	"github.com/ChainSafe/gossamer/internal/log"
+	cfg "github.com/ChainSafe/gossamer/config"
+	"github.com/ChainSafe/gossamer/dot/state/pruner"
 	"github.com/ChainSafe/gossamer/lib/common"
 	"github.com/ChainSafe/gossamer/lib/genesis"
 	"github.com/ChainSafe/gossamer/lib/runtime/wasmer"
 )
 
 var (
-	// GlobalConfig
-
 	// DefaultName Default node name
-	DefaultName = string("Kusama")
+	DefaultName = "Kusama"
 	// DefaultID Default chain ID
-	DefaultID = string("ksmcc3")
-	// DefaultConfig Default toml configuration path
-	DefaultConfig = string("./chain/kusama/config.toml")
+	DefaultID = "ksmcc3"
 	// DefaultBasePath Default node base directory path
-	DefaultBasePath = string("~/.gossamer/kusama")
+	DefaultBasePath = "~/.gossamer/kusama"
 
-	// DefaultMetricsAddress is the default metrics server listening address.
-	DefaultMetricsAddress = "localhost:9876"
+	// DefaultPrometheusPort is the default metrics server listening address.
```

Comment:
Do we want to hardcode as prometheus address rather than metrics address?

Reply:
Hmm, do you mean `DefaultPrometheusPort` => `DefaultPrometheusAddress`? The flag in substrate is ``--prometheus-port`